kubeflow/mpi-job/prototypes/mpi-job-custom.jsonnet

```diff
@@ -0,0 +1,26 @@
+// @apiVersion 0.1
+// @name io.ksonnet.pkg.mpi-job-custom
+// @description A custom MPI Job.
+// @shortDescription A custom MPI Job that allows you to specify number of replicas and GPUs per replica.
+// @param name string Name to give to each of the components
+// @param replicas number Total number of replicas for the job
+// @param gpusPerReplica number Total number of GPUs per replica
+// @param image string The image to use for the job
```

Comment:
Which image should user use here? Is it publicly available? If it is we can set default image here.
Maybe also set default value for replicas and gpusPerReplica?

Reply:
This is a user's job, so I suppose it should use whatever image the user has built? We can put in the tensorflow benchmark image here as the default, but I'm not sure if that's very sensible.

What would be good defaults for replicas and gpusPerReplica? Set them both to 1?

Reply:
OK, make them optional and added defaults.